data/os/RedHat/CentOS/8.yaml

```diff
@@ -14,82 +14,89 @@ yum::repos:
   AppStream:
     enabled: true
     descr: 'CentOS-$releasever - AppStream'
-    mirrorlist: 'http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=AppStream&infra=$infra'
+    baseurl: 'https://vault.centos.org/$contentdir/$releasever/AppStream/$basearch/os/'
+    mirrorlist: absent
     gpgcheck: true
     gpgkey: "file:///etc/pki/rpm-gpg/RPM-GPG-KEY-centosofficial"
-    target: '/etc/yum.repos.d/CentOS-AppStream.repo'
+    target: '/etc/yum.repos.d/CentOS-Linux-AppStream.repo'
```

Comment:
is it a breaking change if we change the target? the old files could still exist and then we've the repo declared twice?

Reply:
It is not, since this parameter is still not handled properly by yumrepo resource in any puppet versions
https://tickets.puppetlabs.com/browse/MODULES-7422

But, this is the state where the final Centos update is in
```
$ rpm -qf /etc/yum.repos.d/CentOS-Linux-AppStream.repo 
centos-linux-repos-8-3.el8.noarch

```